### ISSUE TITLE
Several source files are missing from CMakeList.txt

### ISSUE THREAD

**Issue by tuturha** [noloader/cryptopp-cmake#42]
Hi,

I'm installing cryptopp using CMake on my windows computer, it seems that in the latest version (version 7.00) of cryptopp there isn't anymore some previous sources files. So the execution of cmake command gives an error at the moment it try to add to the executable (line 1085):

    CMake Error at CMakeLists.txt:1085 (add_executable):
    Cannot find source file:
    C:/Program Files/CryptoPP/cryptopp-CRYPTOPP_7_0_0/bench3.cpp

    Tried extensions .c .C .c++ .cc .cpp .cxx .cu .m .M .mm .h .hh .h++ .hm
    .hpp .hxx .in .txx

   CMake Error at CMakeLists.txt:1085 (add_executable):
   No SOURCES given to target: cryptest

It refers to the cryptopp_SOURCES_TEST whee the files.cpp are added :

- bench3.cpp (line 522)
- validat5/6/7/8/9/10.cpp (lines 528-533)
- regtest4.cpp (line 537)

When i commented those lines, the cmake execution worked. Maybe it needs to be updated.

**Comment by noloader** [comment]
I don't think we have a `sources.cpp` file. Are you using something provided by someone else?

Also, the tip of cryptopp-cmake follows the tip of weidai/cryptopp. If you are using a released version of the library, then you should grab the CMake files from [releases](https://github.com/noloader/cryptopp-cmake/releases). The releases are the snapshots in time. Both cryptopp-cmake and weidai/cryptopp use the same numbering scheme.

**Comment by tuturha** [comment]
I named "sources.cpp" because multiples sources files are missing. I listed all of them in my post.
The missing files are :
- bench3.cpp
- validat5.cpp
- validat6.cpp
- validat7.cpp
- validat8.cpp
- validat9.cpp
- validat10.cpp
- regtest4.cpp

By the way, further in the configuration of my project i noticed that in the  cmake files it refers a "cryptopp-shared.lib" in the lib folder after building, but there is no such file. There is only the static library. The shared one is in the bin directory with the extension .dll.

I resolved that problem by altering a cmake file (even if i'm not supposed to do that, i know).
